components/brave_ads/browser/notification_helper_android.cc

```diff
@@ -28,6 +29,11 @@ bool NotificationHelperAndroid::IsNotificationsAvailable() const {
 // Starting in Android 8.0 (API level 26), all notifications must be
 // assigned to a channel.
 bool NotificationHelperAndroid::IsBraveAdsChannelEnabled() const {
+  char osVersion[PROP_VALUE_MAX+1];
+  int os_version = __system_property_get("ro.build.version.release", osVersion);
```

Comment:
should we not be using `ro.build.version.sdk` as SDKs define the availability of interfaces?

Reply:
@yachtcaptain23 @tmancey I would use `base::SysInfo::OperatingSystemVersionNumbers` to get OS version number, here is an example https://github.com/brave/browser-android-tabs/blob/base-75.0.3770.89/media/base/android/media_drm_bridge.cc#L256